src/routes/docs/tooling/appwriter/+page.markdoc

```diff
@@ -40,9 +40,17 @@ The keyboard comes with Mac layout keys pre-installed for macOS which can also b
 |Switch support|Hot-swappable switches with 3 & 5 pin|
 |Rollover|N-key rollover|
 
-# How to change the backlight {% #how-to-change-the-backlight %}
+# Controlling the backlighting {% #controlling-the-backlighting %}
```

Comment:
Wouldn't be "Control the backlighting" be correct?

Reply:
Either or really.

"Control the backlighting" is more direct and instructional, suggesting that the section will tell you how to do it, whereas, "Controlling the backlighting" is more descriptive, suggesting that the section will discuss the topic of managing the backlighting.

I'll change it to control